grouper/fe/handlers/audits_complete.py

```diff
@@ -31,37 +31,37 @@ def post(self, audit_id):
             if argument.startswith("audit_"):
                 edges[int(argument.split("_")[1])] = self.request.arguments[argument][0].decode()
 
-        for member in audit.my_members():
-            if member.id in edges:
+        for ami in get_group_audit_members_infos(self.session, audit.group):
+            if ami.audit_member_obj.id in edges:
                 # You can only approve yourself (otherwise you can remove yourself
                 # from the group and leave it ownerless)
-                if member.member.id == self.current_user.id:
```

Comment:
Well I'm happy with this change to remove `member.member` being a thing if nothing else